### PR TITLE
Add: build.ps1 Script with same functions as build.sh

### PR DESCRIPTION
**Pull request by Boxersteavee**
I made a ps1 file with commands that does the same thing as the build.sh script already in the repo.

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+echo "Building the frontend..."
+cd frontend
```
Comment (zleyyij) [inline build.ps1:2]: Please prefer pure powershell
```suggestion
Set-Location frontend
```

File: build.ps1 [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+npm i
+npm run build
+cd ..
+echo "Copying frontend files..."
+mkdir target\web
+xcopy frontend\build target\web /s /e
```
Comment (zleyyij) [inline build.ps1:8]: This should preferably be done in powershell with `Move-Item` to keep things consistent.

File: build.ps1 [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+echo "Building the backend..."
+cd backend
+cargo build --release
+echo "Copying backend files..."
+mkdir target\hyde
```
Comment (zleyyij) [inline build.ps1:14]: Same thing goes for all of this, please prefer native powershell commands over aliases or windows binaries

**Review by zleyyij — CHANGES_REQUESTED** [review]

**Comment by Boxersteavee** [comment]
I'll tweak and redo it properly with normal powershell later. I kinda rushed this and mostly copied from the bat file I made (that then didn't work because batch files are a bitch to make work)

**Comment by Boxersteavee** [comment]
Ignore latest commit. Somehow the readme got removed?

**Review by zleyyij — APPROVED** [review]

**Pull request merged**